www/js/app.js

```diff
@@ -10,129 +10,133 @@ angular.module('pvta', ['ionic', 'ngCordova', 'pvta.controllers', 'angularMoment
-    // Hide the accessory bar by default (remove this to show the accessory bar above the keyboard
-    // for form inputs)
+                // Hide the accessory bar by default (remove this to show the accessory bar above the keyboard
+                // for form inputs)
     if (window.cordova && window.cordova.plugins.Keyboard) {
       cordova.plugins.Keyboard.hideKeyboardAccessoryBar(true);
       cordova.plugins.Keyboard.disableScroll(true);
     }
   });
-  /******************************************************
-  * Set the status bar color to our app's color.
-  *********************************************************
-  * Only after the device has specifically told us it's ready
-  * may we access the global StatusBar object.
-  * **************************************************/
+        /******************************************************
+         * Set the status bar color to our app's color.
+         *********************************************************
+         * Only after the device has specifically told us it's ready
+         * may we access the global StatusBar object.
+         * **************************************************/
   document.addEventListener('deviceready', onDeviceReady, true);
   function onDeviceReady () {
     StatusBar.backgroundColorByHexString('#387ef5');
   }
 })
 
-.config(function ($stateProvider, $urlRouterProvider, $ionicFilterBarConfigProvider) {
+.config(function ($stateProvider, $urlRouterProvider) {
   $stateProvider
 
-  .state('app', {
-    url: '/app',
-    abstract: true,
-    templateUrl: 'templates/menu.html',
-    controller: 'AppController'
-  })
+        .state('app', {
+          url: '/app',
+          abstract: true,
+          templateUrl: 'templates/menu.html',
+          controller: 'AppController'
+        })
 
 
-  .state('app.my-buses', {
-    url: '/my-buses',
-    views: {
-      'menuContent': {
-        templateUrl: 'templates/mybuses.html',
-        controller: 'MyBusesController'
-      }
+.state('app.my-buses', {
+  url: '/my-buses',
+  views: {
+    'menuContent': {
+      templateUrl: 'templates/mybuses.html',
+      controller: 'MyBusesController'
     }
-  })
-  .state('app.search', {
-    url: '/search',
-    views: {
-      'menuContent': {
-        templateUrl: 'templates/search.html',
-        controller: 'SearchController'
-      }
+  }
+})
+.state('app.search', {
+  url: '/search',
+  views: {
+    'menuContent': {
+      templateUrl: 'templates/search.html',
+      controller: 'SearchController'
     }
-  })
-    .state('app.vehicle', {
-      url: '/vehicles/:vehicleId/:route',
-      params: {
-      // For whatever reason,
-      // squash: true means that
-      // :route is an optional param
-        route: {squash: true}
-      },
-      views: {
-        'menuContent': {
-          templateUrl: 'templates/vehicle.html',
-          controller: 'VehicleController'
-        }
-      }
-    })
+  }
+})
+.state('app.vehicle', {
```

Comment:
This basically just undoes what's already in Master.  I feel like you haven't pulled/merged your branch with master properly?